src/components/entity/ha-chart-base.html

```diff
@@ -335,8 +370,46 @@
           this._drawLegend();
         }
         this.resizeChart();
+        this.updateZoomlimits();
       }
     }
+    updateZoomlimits() {
+      if (!this._chart) return;
+      if (this.isTimeline) return;
+
+      const buffer = {
+        x: { min: null, max: null },
+        y: { min: null, max: null }
+      };
+      Object.entries(this._chart.scales).forEach(([name, scale]) => {
+        if ({}.hasOwnProperty.call(this._chart.scales, name)) {
```

Comment:
Why doing this fancy? Aren't `scales` just objects and not complicated objects? 

```js
if (name in this._chart.scales) {

}
```

Reply:
No reason, just how I got eslint to stop complaining :smiley: 
- replaced both `{}.hasOwnProperty` with `(x in y)`